.github/workflows/upload-release-artifact.yml

```diff
@@ -0,0 +1,53 @@
+name: Upload release artifact
+
+on:
+  release:
+    types: [ published ]
+
+env:
+  PROJECT: rest-list-parameter
```

Comment:
This should also have a different name

Reply:
same as above, sorry! (note to self: stop copying things everywhere without proper second pass of updating the values)